modules/appeals_api/appeals_api.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.rdoc']
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'rails', '~> 5.2.3'
+  s.add_dependency 'rails', '>= 5.2.3'
```

Comment:
This requirement was stopping the bundle update from working, so I started to change it to `'~> 6.0.2'` but then realized that whoever came along next would have the same issue.  I think I'm more on the side of stripping the version completely since they are part of vets-api.  But I also see the logic in making it match what was in the main bundle.

Reply:
I'm not sure there's a need for *any* rails dependency is there? i mean clearly this is only going to be used in this rails app, but its a bit redundant and an extra step at every upgrade. 

I would guess at one point these said `~> 4.2` back in the beginning :)

Reply:
>  I think I'm more on the side of stripping the version completely since they are part of vets-api.

> I'm not sure there's a need for any rails dependency is there?

even better idea.